admin_guide/cluster_metrics.adoc

```diff
@@ -226,9 +235,29 @@ items:
               args:
                 - "-source=kubernetes:https://kubernetes.default.svc?auth=&insecure=true&useServiceAccount=true"
```

Comment:
The one we are using for the origin-metrics is:
--source=kubernetes:${MASTER_URL}?useServiceAccount=true&kubeletHttps=true&kubeletPort=10250

The template we have for that is available here: https://github.com/openshift/origin-metrics/blob/master/deployer/templates/heapster.yaml